extensions/mssql/package.json

```diff
@@ -367,6 +367,16 @@
           "type": "boolean",
           "description": "%mssql.ignorePlatformWarning%",
           "default": false
+        },
+        "mssql.useKeytarCredential": {
+          "type": "boolean",
+          "description": "%mssql.useKeytarCredential%",
```

Comment:
This should really be a one time, one way thing where the user chooses to migrate at some point and then from then on we only use keytar. So because of that I don't think a setting configuration is appropriate here since that means it'll show up even well after the user has done the migration and should no longer be able to go back.